processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -97,6 +107,16 @@ func (ctdp *cumulativeToDeltaProcessor) shutdown(context.Context) error {
 	return nil
 }
 
+func (ctdp *cumulativeToDeltaProcessor) convertMetric(metricName string) bool {
```

Comment:
```suggestion
func (ctdp *cumulativeToDeltaProcessor) shouldConvertMetric(metricName string) bool {
```

This method name should convey that it is a predicate and not a mutator.